Modules/QuestieAnnounce.lua

```diff
@@ -77,13 +99,42 @@ function QuestieAnnounce:ItemLooted(text, notPlayerName, _, _, playerName)
         end
 
         if startQuestId then
-            if (not UnitInParty("player")) or (not Questie.db.char.questAnnounce) then
-                _QuestieAnnounce:AnnounceSelf(startQuestId, itemId)
+            if _QuestieAnnounce:AnnounceEnabledAndPlayerInChannel() then
+                QuestieAnnounce:AnnounceQuestItemLootedToChannel(startQuestId, itemId)
                 return
             end
 
-            QuestieAnnounce:AnnounceParty(startQuestId, "item", itemId)
+            _QuestieAnnounce:AnnounceSelf(startQuestId, itemId)
         end
     end
 end
 
+function QuestieAnnounce:AcceptedQuest(questId)
+    if (_QuestieAnnounce:AnnounceEnabledAndPlayerInChannel()) and Questie.db.char.questAnnounceAccepted then
+        -- TODO test if this works if quest hasn't yet been accepted by other party members
```

Comment:
Did you have the time to complete this TODO?

Reply:
Ah, I remember. I wanted to use `QuestieLink:GetQuestHyperLink(questId)` in the following line, but I got the error message ```
SendChatMessage(): Invalid escape code in chat message
```
so I reverted back to `QuestieLink:GetQuestLinkString`.
I also observed I cannot click the accepted quest link in my own messages, while party members can. Intriguingly the same method produces a clickable link for abandoned quests. I strongly believe there is something odd in the link syntax.
For know I'd consider this a minor issue.